docs/modules/ROOT/pages/configuration.adoc

```diff
@@ -739,11 +739,11 @@ Running `rubocop --autocorrect --disable-uncorrectable` will
 create comments to disable all offenses that can't be automatically
 corrected.
 
-Do not write anything other than cop name in the disabling comment. E.g.:
+Add a comment to explain why the cop has been disabled E.g.:
```

Comment:
I'd change the wording to something like "You can also...", so it sounds like a suggestion. I think you removed the `.` before e.g. by accident.

Reply:
👍🏻 I updated the language to be `You can also include...` and re-added the period.